pennylane/_device.py

```diff
@@ -363,6 +365,25 @@ def var(self, observable, wires, par):
         """
         raise NotImplementedError
 
+    @abc.abstractmethod
```

Comment:
At least until the plugins are updated, I would suggest removing the abstract method decorator. This way, existing plugins will still work and load, but any attempt by a user to execute a QNode that returns samples will result in a `NotImplementedError`.

Once all plugins support samples and variance, we can make this (and `var`) an abstract method.

Note: let's update the `NotImplementedError` here and in `var` to
```python
raise NotImplementedError("Returning samples from QNodes not currently supported by {}".format(dev.short_name))
```

Reply:
Will be removed / included in the next commit!